diff --git a/node_modules/react-native-css-interop/dist/runtime/native/render-component.js b/node_modules/react-native-css-interop/dist/runtime/native/render-component.js
--- a/node_modules/react-native-css-interop/dist/runtime/native/render-component.js
+++ b/node_modules/react-native-css-interop/dist/runtime/native/render-component.js
@@ -131,8 +131,20 @@ function stringify(object) {
         }
         seen.add(value);
         const newValue = Array.isArray(value) ? [] : {};
-        for (const entry of Object.entries(value)) {
-            newValue[entry[0]] = replace(entry[0], entry[1]);
+        for (const key of Object.keys(value)) {
+            const descriptor = Object.getOwnPropertyDescriptor(value, key);
+            if (!descriptor)
+                continue;
+            if ("value" in descriptor) {
+                newValue[key] = replace(key, descriptor.value);
+            }
+            else {
+                newValue[key] = descriptor.get
+                    ? descriptor.set
+                        ? "[Getter/Setter]"
+                        : "[Getter]"
+                    : "[Setter]";
+            }
         }
         seen.delete(value);
         return newValue;
diff --git a/node_modules/react-native-css-interop/src/runtime/native/render-component.tsx b/node_modules/react-native-css-interop/src/runtime/native/render-component.tsx
--- a/node_modules/react-native-css-interop/src/runtime/native/render-component.tsx
+++ b/node_modules/react-native-css-interop/src/runtime/native/render-component.tsx
@@ -251,8 +251,19 @@ function stringify(object: any) {
 
       const newValue: any = Array.isArray(value) ? [] : {};
 
-      for (const entry of Object.entries(value)) {
-        newValue[entry[0]] = replace(entry[0], entry[1]);
+      for (const key of Object.keys(value)) {
+        const descriptor = Object.getOwnPropertyDescriptor(value, key);
+        if (!descriptor) continue;
+
+        if ("value" in descriptor) {
+          newValue[key] = replace(key, descriptor.value);
+        } else {
+          newValue[key] = descriptor.get
+            ? descriptor.set
+              ? "[Getter/Setter]"
+              : "[Getter]"
+            : "[Setter]";
+        }
       }
 
       seen.delete(value);
